pkg/status/endpoints/endpoints.go

```diff
@@ -0,0 +1,34 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+// Package endpoints fetch information needed to render the 'endpoints' section of the status page.
+package endpoints
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/config/utils"
+)
+
+// PopulateStatus populates the status stats
+func PopulateStatus(stats map[string]interface{}) {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏You could have kept `getEndpointsInfos` and just have `PopulateStatus` check the return value and error